modules/installation-overview.adoc

```diff
@@ -7,50 +7,148 @@
+
+These two basic types of {product-title} clusters are
+Installer Provisioned Infrastructure (IPI) clusters and
+and User Provisioned Infrastructure (UPI) clusters.
+
+You use the same installation program to deploy both types of clusters. The main
+assets generated by the installation program are the Ignition configurations for the
+bootstrap, master, and worker machines. With these three configurations and correctly
+configured infrastructure, you can start an {product-title} cluster.
+
+[id="available-platforms-{context}"]
+== Available platforms
+
+In {product-title} {product-version}, you can complete IPI installations on
+only Amazon Web Services (AWS).
+
+You can complete UPI installations on the following platforms:
+
+* AWS
+////
+* vSphere
+* Bare metal
+////
+
+[id="installation-process-{context}"]
+== Installation process
+
+When you install a {product-title} cluster, you download the
+installer from link:https://cloud.openshift.com/clusters/install[the OpenShift start page].
+This site manages:
+
+* REST API for accounts
+* Registry tokens, which are the pull secrets that you use to obtain the required
+components
+* Cluster registration, which associates the cluster identity to your Red Hat
+account to facilitate the gathering of usage metrics
+
+In {product-title} 4, the installation program is a Go binary file that performs a
+series of file transformations on a set of assets. The way you interact with the
+installation program differs depending on your installation type.
+
+* When you use IPI, you delegate the infrastructure bootstrapping and provisioning
+to the installation program
+instead of doing it yourself. Because you do not use the installation program to upgrade or
+update your cluster, if you do not customize your cluster, you run the
+installer only once. If you customize your cluster, you run the installation
+program to generate the required configuration files and then run it again to
+deploy your cluster.
+
+* When you use UPI, you must
+provide all of the cluster infrastructure and resources, including the
+bootstrap node. You must run the installation program to generate the required
+configuration files and then run the installation program again to deploy a cluster
+to your configured infrastructure.
+
+You use three sets of files during installation: an installation configuration
+file, Kubernetes manifests, and Ignition configurations for your machine types.
+
+[IMPORTANT]
+====
+It is possible to modify Kubernetes and the Ignition Configs that control
+the underlying {op-system-first} operating system during installation. However,
+no validation is available to confirm the suitability of any modifications that
+you make to these objects. If you modify these objects, you might render
+your cluster non-functional. Because of this risk, modifying Kubernetes and
+Ignition Configs is not supported unless you are following documented procedures
+or instructed to do so by Red Hat support.
```

Comment:
nit: "are following" doesn't match "instructed" for the "or".  Maybe "were instructed"?

Reply:
Without a second auxiliary verb, the "are" applies to instructed, too, but I'll make it explicit for clarity. Thanks!